model/metabolites.tsv

```diff
@@ -2735,6 +2735,7 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM01638e"	"MAM01638"	"ala__D"	"C00133"	"HMDB01310"	"CHEBI:15570"	"71080"	""	""	""	"ala_D"	"MNXM156"	"m01638s"	"m01638s"
 "MAM01639c"	"MAM01639"	"damp"	"C00360"	""	"CHEBI:17713"	"12599"	""	""	"HC00304"	"damp"	"MNXM432"	"m01639c"	"m01639c"
 "MAM01639l"	"MAM01639"	"damp"	"C00360"	""	"CHEBI:17713"	"12599"	""	""	"HC00304"	"damp"	"MNXM432"	"m01639l"	"m01639l"
+"MAM01639n"	"MAM01639"	"damp"	"C00360"	""	"CHEBI:17713"	"12599"	""	""	"HC00304"	"damp"	"MNXM432"	"m01639n"	"m01639n"
```

Comment:
Unless somebody has verified that this metabolite has previously existed in HMR2, the references should be deleted.
```suggestion
"MAM01639n"	"MAM01639"	"damp"	"C00360"	""	"CHEBI:17713"	"12599"	""	""	"HC00304"	"damp"	"MNXM432"	""	""
```

Reply:
I agree